Droid/Resources/layout/ConcactCardView.axml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:card_view="http://schemas.android.com/apk/res-auto"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="fill_parent"
+    android:layout_height="wrap_content">
+    <android.support.v7.widget.CardView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        card_view:cardElevation="4dp"
```

Comment:
https://github.com/Karumi/KataSuperHeroesAndroid/blob/master/app/src/main/res/layout/super_hero_detail_activity.xml#L58 + https://github.com/Karumi/KataSuperHeroesAndroid/blob/master/app/src/main/res/values/dimens.xml + https://github.com/Karumi/KataSuperHeroesAndroid/blob/master/app/src/main/res/values/colors.xml

If you ask me, it's completely useless unless you have a designer doing components but it's standard in Android to remove all dimens/colors from layouts